### PR TITLE
Forgot password feature added in front end

### PR DESCRIPTION
**Pull request by Shubham-Kumar-2000**
## Issue that this pull request solves

 Closes: #475 

## Proposed changes
Deliverable:
Add a forgot password form having following field

Email Id
Submit Button
After clicking on Submit button, a new page should come up saying An invitation link has been sent on your registered email, please check your inbox for further steps Along with two buttons 1. To go to login page 2. To go to home page

### Brief description of what is fixed or changed
Above page was successfully created and tested.

There was a bug in \frontend\src\components\util\Button\Button.jsx
That it wasn't considering the className from props, This issue was also fixed.

Added a Forgot Password link in Login Page.

My code Generates some unused variable warning because backend is not yet made and connected.
Code for backend connection also has been written and just required to be uncommented when  backend is ready.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

image

image

## Other information

NA

### CODE REVIEW

File: frontend/src/pages/ForgotPassword/ForgotPassword.module.scss [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  margin: 0 auto;
+  margin-bottom: 30px !important;
+}
+
+.submit-btn-text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.back-button {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/ForgotPassword/ForgotPassword.module.scss:141]: please leave one line space here

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @Shubham-Kumar-2000 The UI looks preety good to me. Just need minor fixes :point_down: 

Please handle the email-validation properly and show-up a clean error message to the user
Screenshot from 2021-03-09 15-53-06

**Comment by Shubham-Kumar-2000** [comment]
I have made the required Changes and pushed the code.
Please check @Kajol-Kumari

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for the quick fix @Shubham-Kumar-2000 
LGTM :+1:

**Pull request merged**